editor/store/reducer.js

```diff
@@ -421,14 +422,13 @@ export function blockSelection( state = {
 				start: action.uid,
 				end: action.uid,
 				focus: action.config || {},
+				next: null,
 			};
 		case 'INSERT_BLOCKS':
 			return {
 				...state,
-				start: action.blocks[ 0 ].uid,
-				end: action.blocks[ 0 ].uid,
-				focus: {},
-				isMultiSelecting: false,
+				// We must wait for Editable to initialise.
+				next: action.blocks[ 0 ].uid,
```

Comment:
Not sure I understand here, you're adding a `next` saying that this block will be focused next but for me `focus: {}` means the same thing?

So wondering why the `focus` prop is not passed at the same moment to the block without this change?

Reply:
The `focus` props for the previous block will be taken away, which will cause the area to blur.

Reply:
So you want to focus before blur right? Maybe we can just delay the `.blur` call?

Reply:
How do we do that across instances? I guess we could maybe use something like `scrollPosition`. `scrollPosition || blur`.

Reply:
Actually, I wonder if this `.blur` called is needed at all, might be good to check the commit adding it.

Reply:
The `.blur()` call was added here https://github.com/WordPress/gutenberg/pull/683#issuecomment-299548842 

Where the reason some more "logical" than based on a use case (cc @aduth). If removing the blur fixed the mobile issue, I'd vote for that :)

Reply:
Calling blur might be bad in any case, because the focus should always move somewhere else. This will just move it to the body element.